.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement, Impact/Severity: Low, Priority: Low, waiting for collaborator
+  review'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
this question and the next feel redundant when we already ask user story questions.
IMO this question feels similar to why they want the feature

Reply:
Remove this one?

Reply:
yes please